### PR TITLE
custom hide or show header in response

### PR DESCRIPTION
**Pull request by Panmax**
#### Changes proposed in this pull request:
 - custom hide or show header in response

### CODE REVIEW

File: spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java [hunk 1]
```diff
@@ -62,6 +62,7 @@
     private Map<String, List<Policy>> policyList = Maps.newHashMap();
     private boolean behindProxy;
     private boolean enabled;
+    private boolean showHeader = true;
```
Comment (marcosbarbero) [inline spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java]: I think a name like `addResponseHeaders` would be cleaner on what this property does, WDYT?

Comment (Panmax) [reply spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java]: you're right

**Comment by marcosbarbero** [comment]
Thanks for your contribution @Panmax

**Comment by marcosbarbero** [comment]
@Panmax it's available on `v2.2.3.RELEASE`

**Comment by Panmax** [comment]
> @Panmax it's available on `v2.2.3.RELEASE`

thank you.

**Pull request merged**